x-pack/plugins/infra/public/components/logging/log_analysis_setup/subscription_splash_content.tsx

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiPage,
+  EuiPageBody,
+  EuiPageContent,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+  EuiTitle,
+  EuiText,
+  EuiButton,
+  EuiButtonEmpty,
+  EuiImage,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { LoadingPage } from '../../loading_page';
+
+import { useKibana } from '../../../../../../../src/plugins/kibana_react/public';
+import { euiStyled } from '../../../../../observability/public';
+import { useTrialStatus, TrialStatusLoadState } from '../../../hooks/use_trial_state';
+
+export const SubscriptionSplashContent: React.FC = () => {
+  const { services } = useKibana();
+  const { loadState, isTrialAvailable } = useTrialStatus();
+
+  if (loadState === TrialStatusLoadState.Loading) {
```

Comment:
That's a fair point. My reasoning to export it as an enum was to also control the error state using only one variable.

So instead of...

```ts
{
  isLoading: true | false,
  isError: true | false,
}  
```

...I only return one variable that contains the necessary state.

I don't mind changing it to `isLoading` to keep the consistency, but I think there's value in having a conversation afterwards about how we want to expose network state.